packages/ui/app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const path = require("path");
+const cookieParser = require("cookie-parser");
```

Comment:
@rishabh3112 why we're using cookies?

Reply:
Its a express generator template. Will update after foundation is merged and we are working on server !!

Maybe we can use cookies too!! for storing previous responses by user for init